README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
+- No ads, trackers, subscriptions, etc.
+- Maintaining and developing the app can be an hobby: low (or zero) fees, low architectural complexity, supported by the community
+- User data is owned by and hosted by the respective user
```

Comment:
"Hosted" is the wrong word here. I think "owned" is enough. Can we change it to:

User data is owned by each user, and is always available to them. This data is in an open format.

Reply:
Updated